tests/invalid_models_tests/test_relative_fields.py

```diff
@@ -115,6 +116,11 @@ class ThroughModel(models.Model):
                 obj=field,
                 id='fields.W340',
             ),
+            DjangoWarning(
+                'db_comment has no effect on ManyToManyField.',
+                obj=field,
+                id='fields.W345',
+            ),
```

Comment:
This test is failing when I run the test suite locally. Here's the error message:
```
FAIL: test_many_to_many_with_useless_options (invalid_models_tests.test_relative_fields.RelativeFieldTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jaredchung/Documents/workspace/django-dbcomment/django/tests/invalid_models_tests/test_relative_fields.py", line 113, in test_many_to_many_with_useless_options
    self.assertEqual(ModelM2M.check(), [
AssertionError: Lists differ: [<War[16 chars]msg='SQLite does not support db_comment on col[759 chars]45'>] != [<War[16 chars]msg='null has no effect on ManyToManyField.', [597 chars]43'>]

First differing element 0:
<Warn[15 chars]msg='SQLite does not support db_comment on col[89 chars]163'>
<Warn[15 chars]msg='null has no effect on ManyToManyField.', [83 chars]340'>

First list contains 1 additional elements.
First extra element 4:
<Warning: level=30, msg='db_comment has no effect on ManyToManyField.', hint=None, obj=<django.db.models.fields.related.ManyToManyField: m2m>, id='fields.W345'>

Diff is 1758 characters long. Set self.maxDiff to None to see it.
```